pkg/deployment/resources/pod_creator_arangod.go

```diff
@@ -66,7 +66,8 @@ type MemberArangoDPod struct {
 	resources        *Resources
 	imageInfo        api.ImageInfo
 	autoUpgrade      bool
-	id               string
+	args             []string
```

Comment:
Why providing it as static list? We can generate it

Reply:
changed